apps/backend/src/routes/signup/+page.svelte

```diff
@@ -0,0 +1,243 @@
+
+      <!-- Signup Form - Corporate Email Input -->
+      <div>
+        <label for="email" class="form-control w-full max-w-2xl">
+          <div class="label">
+            <span class="label-text">Corporate Email Address</span>
+            <span class="text-red-600 label-text-alt">*</span>
+          </div>
+          <input
+            id="email"
+            type="text"
+            name="email"
+            placeholder="Name@example.com"
+            on:change={handleChange}
+            bind:value={$form.email}
+            class="
+              input
+              input-bordered
+              w-full max-w-2xl
+              focus:ring-secondary
+              focus:border-secondary
+              duration-300
+              focus:outline-none
+            " />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.email}</span>
+        </div>
+      </div>
+
+      <!-- Signup Form - Password Input -->
+      <div>
+        <label for="password" class="form-control w-full max-w-2xl">
+          <div class="label">
+            <span class="label-text">Password</span>
+            <span class="text-red-600 label-text-alt">*</span>
+          </div>
+          <input
+            id="password"
+            type="password"
+            name="password"
+            placeholder="••••••••"
+            on:change={handleChange}
+            bind:value={$form.password}
+            class="
+              input
+              input-bordered
+              w-full
+              max-w-2xl
+              focus:ring-secondary
+              focus:border-secondary
+              duration-300
+              focus:outline-none
+            " />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.password}</span>
+        </div>
+      </div>
+
+      <!-- Signup Form - Confirm Password Input -->
+      <div>
+        <label class="form-control w-full max-w-2xl">
+          <div class="label">
+            <span class="label-text">Confirm Password</span>
+            <span class="text-red-600 label-text-alt">*</span>
+          </div>
+          <input
+            type="password"
+            name="confirmpass"
+            placeholder="••••••••"
+            on:change={handleChange}
+            bind:value={$form.confirmPassword}
+            class="
+              input
+              input-bordered
+              w-full
+              max-w-2xl
+              focus:ring-secondary
+              focus:border-secondary
+              duration-300
+              focus:outline-none
+            " />
+        </label>
+        <div>
+          <span class="text-red-600">{$errors.confirmPassword}</span>
+        </div>
+      </div>
+
+      <div class="form-control mt-7">
+        <label class="label cursor-pointer">
+          <input
+            type="checkbox"
+            class="checkbox checkbox-secondary mr-5"
+            bind:checked={trigger}
+            on:click={toggleCheckbox} />
```

Comment:
Switch to using `svelte-forms-lib` please

Reply:
Missing `on:change={handleChange}` attribute

Reply:
Added:

```
<input
            type="checkbox"
            class="checkbox checkbox-secondary mr-5"
            bind:checked={$form.tosCheckbox}
            on:change={handleChange} />
```